modules/imgproc/include/opencv2/imgproc.hpp

```diff
@@ -3267,6 +3267,10 @@ CV_EXPORTS float EMD( InputArray signature1, InputArray signature2,
                       int distType, InputArray cost=noArray(),
                       float* lowerBound = 0, OutputArray flow = noArray() );
 
+CV_EXPORTS_AS(EMD) float wrapperEMD( InputArray signature1, InputArray signature2,
```

Comment:
I believe, simple overload should work here:

```.diff
-CV_EXPORTS_AS(EMD) float wrapperEMD(
+CV_EXPORTS_W float EMD(
```

(probably without default value for `lowerBound`)